js/admin/notifications.js

```diff
@@ -22,151 +22,120 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-$(document).ready(function() {
-	var hints = $('.translatable span.hint');
-	if (youEditFieldFor)
-		hints.html(hints.html() + '<br /><span class="red">' + youEditFieldFor + '</span>');
-
-	var html = "";
-	var nb_notifs = 0;
-	var wrapper_id = "";
-	var type = new Array();
+$(document).ready(function () {
+  if (youEditFieldFor) {
+    $('.translatable span.hint').append('<br /><span class="red">' + youEditFieldFor + '</span>')
```

Comment:
```suggestion
    $('.translatable span.hint').append(`<br /><span class="red">${youEditFieldFor}</span>`)
```

Reply:
I prefer to use template-strings but I think this file is not transpiled, it cannot be used if we want to maintain compatibility with some old browsers.

https://caniuse.com/?search=template%20strings

Reply:
Where can I check the browser versions compatibility table to see in the development of the back office and the front for each Prestashop version?

IE 11 is no longer necessary right?

What about android 4.4 and earlier?